conn.go

```diff
@@ -82,6 +82,19 @@ type ConnConfig struct {
 	Partition int
 }
 
+type BatchReadConfig struct {
+	MinBytes       int
+	MaxBytes       int
+	IsolationLevel IsolationLevel
+}
+
+type IsolationLevel int8
+
+const (
+	READ_UNCOMMITTED IsolationLevel = 0
+	READ_COMMITTED   IsolationLevel = 1
```

Comment:
```suggestion
	ReadCommitted   IsolationLevel = 1
```
Let's follow Go naming guidelines ;)